honeybadgermpc/mimc.py

```diff
@@ -0,0 +1,64 @@
+from math import log, ceil
+import logging
+import asyncio
+from honeybadgermpc.mpc import PreProcessedElements, TaskProgramRunner, Subgroup
+from honeybadgermpc.field import GF
+from honeybadgermpc.mixins import BeaverTriple, MixinOpName
+
+# ROUND: iteration time of MiMC encryption function, r = ceil(log(p, 3)) = 161
+ROUND = ceil(log(Subgroup.BLS12_381))
+
+
+async def mimc(context, x, k):
+    # message x \in Fp, secret key k \in Fp
```

Comment:
I think using a docstring comment here would be better, as well as including the link to the algorithm.
```suggestion
"""
MiMC block cipher encryption encrypts message x with secret key k,
where x and k are elements of F_p
See: https://eprint.iacr.org/2016/542.pdf
"""
```

Reply:
Resolved.